Makefile

```diff
@@ -252,11 +252,10 @@ endif
 
 ifeq ($(UNAME), FreeBSD)
 STRIP = strip -s $(BUILD_DIR)/$(LIBNAME)
-## Using spinlocks to avoid recursive locks contentions with calloc
-USE_SPINLOCK = -DUSE_SPINLOCK=1
-## Once FreeBSD 13.1 becomes the minimal non EOL version
-## it can be enabled
-## SCHED_GETCPU = -DSCHED_GETCPU
+USE_SPINLOCK = -DUSE_SPINLOCK=0
+## The issue comes only with thread support
```

Comment:
What issue?

Reply:
the tests failing.

Reply:
Can you leave a more descriptive comment?